x-pack/plugins/ml/server/routes/schemas/results_service_schema.ts

```diff
@@ -52,3 +52,14 @@ export const partitionFieldValuesSchema = schema.object({
   earliestMs: schema.number(),
   latestMs: schema.number(),
 });
+
+export const getCategorizerStatsSchema = schema.nullable(
+  schema.object({
+    partitionByValue: schema.maybe(schema.string()),
```

Comment:
Could you add some TSDocs style comments in for all the fields in the new schemas you've added to this file as these will be used in the docs we generate for our Kibana API. For example, it isn't clear to me what `fieldToBucket` should be in the `getStoppedPartitionsSchema`.

Reply:
Updated here https://github.com/elastic/kibana/pull/75061/commits/7d8c39b0fc8d740f237a4fe9f3c4ef6644c5a4ec and here https://github.com/elastic/kibana/pull/75061/commits/3e7de3ab7172b7108e4ca088622a6b49758532ee